src/hotspot/share/opto/library_call.cpp

```diff
@@ -1032,17 +1032,17 @@ bool LibraryCallKit::inline_preconditions_checkIndex(BasicType bt) {
                   Deoptimization::Action_make_not_entrant);
   }
 
+  if (stopped()) {
```

Comment:
Hi @sviswa7, comments over both the short-circuiting conditions will help in better code readability.

Reply:
@jatin-bhateja Added comments, please take a look.